includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -627,6 +655,35 @@ public function register_tag() {
 		$tag->register();
 	}
 
+	/**
+	 * Checks if the current user is a non-Site Kit user.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool True if the current user is a non-Site Kit user, false otherwise.
+	 */
+	private function is_non_sitekit_user() {
+		return ! ( current_user_can( Permissions::VIEW_SPLASH ) || current_user_can( Permissions::VIEW_DASHBOARD ) );
+	}
+
+	/**
+	 * Enqueues assets for non-Site Kit users on block editor pages.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return void
+	 */
+	private function enqueue_assets_for_non_sitekit_user_on_block_editor_page() {
```

Comment:
While we're at it, I feel we can improve the naming of this method. WDYT?

```suggestion
	/**
	 * Enqueues block editor assets for non-Site Kit users.
	 *
	 * @since n.e.x.t
	 *
	 * @return void
	 */
	private function enqueue_block_editor_assets_for_non_sitekit_user() {
```